### DISCUSSION TITLE
IRAF 2.16.1/ no cl.e binary after installing IRAF

### DISCUSSION THREAD

**Discussion by hiraga13** [iraf-community/iraf#187] [Q&A] [answered]
![IMG_20210725_145538.jpg](https://user-images.githubusercontent.com/87889357/126892323-358920bb-1a5c-4d23-b30c-23f4b774798b.jpg)
I've been trying install iraf and x11iraf followed those link: 
https://iraf-community.github.io/install.html
https://www.astr.tohoku.ac.jp/~akhlaghi/irafinstall.html
But the problem is: when I opened xgterm and upwarm then cl but it said there was no cl.e binary found.
What should I do to solve this?
Thank you.

**Comment by olebole** [comment, marked as answer]
Could you add the build log please? You can also check yourself there whether `cl.e` resp. `ecl.e` was not built.

**Reply by hiraga13** [reply]
Thank you for your response. I started to find the build log with journal.ctl command but seemed like not what I'd been looking for.
Could you recommend me better way to get build log?

**Reply by olebole** [reply]
If you followed the [installation procedure](https://iraf-community.github.io/install.html):

> ## Compile the Sources
> […]
> Now you can configure the system for the proper architecture and build:
>
>     $ make <arch>
>     $ make sysgen 2>&1 | tee build.log

then you find the build log in the file `build.log`.

**Reply by hiraga13** [reply]
Thank you. Actually I re-install iraf and found out that set wrong path from step 
```
$ export PATH=/path/to/iraf/bin/:$PATH
```
The old path was usr/local/bin and that made the cl.e couldn't be found
I setted the new one as iraf/iraf and got the good result.
![IMG_20210727_124559.jpg](https://user-images.githubusercontent.com/87889357/127102035-4b3f7e21-8595-4f5f-b5ac-2ad0f86f41d0.jpg)
Thank you
